src/services/cache.ts

```diff
@@ -130,9 +130,12 @@ export class Cache {
     return values
   }
 
+  // node-cache returns a timestamp of when the time is going to expire, while
+  // redis returns an integer meaning remaining seconds, this converts the
+  // node-cache ttl result to remaining seconds
   private getLocalTTL(key: string) {
     const localTTL = this.local.getTtl(key) || 0
     // Gets time difference in seconds
-    return localTTL > 0 ? localTTL : (localTTL - new Date().getTime()) / 1000
+    return localTTL <= 0 ? localTTL : (localTTL - new Date().getTime()) / 1000
```

Comment:
even tho it looks better like this
```suggestion
    return localTTL > 0 ? (localTTL - new Date().getTime()) / 1000 : localTTL
```